modules/installation-overview.adoc

```diff
@@ -179,7 +179,7 @@ the cluster to the infrastructure that you provided.
 If you do not use infrastructure that the installation program provisioned, you must manage
 and maintain the cluster resources yourself, including:
 
-* The control plane and compute machines that make up the cluster
+* The underlying compute infrastructure
```

Comment:
@abhinavdahiya, do you have an opinion on this suggested terminology change?

Reply:
imo this is confusing as users might think the compute machines for cluster are user managed but the control plane is installer created/managed. I like the previous one over this suggested change as it was more explicit

Reply:
I agree. Thank you!

Reply:
Look at this documentation from this perspective. 

Customer Question: 
"When using user-provisioned infrastructure installation you do not use infrastructure that the installation program provisioned but you must manage and maintain the cluster resources yourself, including: what?"

If we say, the "The control plane and compute machines that make up the cluster", what does this mean for "maintain the cluster resources yourself", updates, etc? 

By saying the "underlying compute infrastructure", it make it clear to the user, that "When using user-provisioned infrastructure installation you do not use infrastructure that the installation program provisioned but you must manage and maintain the" - "underlying compute infrastructure", not necessarily the series that run on these hosts. 

If you don't like the proposed language, what language do you suggest, to improve on the existing customer confusion? Simply leaving the language as is; might mean we need to re-write the rest of the section.

Reply:
@abhinavdahiya Could you look at Eric's comment? If you don't agree, @kalexand-rh can you close it and the associated BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1854964

Reply:
@sferich888, I had a chat with Abhinav. Would "The hosts for the control plane and compute machines that make up the cluster" be an acceptable clarification?

Reply:
@sreber84 I think it's better, what's your comment? What the objection with the proposed wording?

Reply:
@kalexand-rh that sounds OK to me and I'm fine with applying this change.